### PR TITLE
Rename master branch to main

### PR DESCRIPTION
**Pull request by hekota**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ For more information, see the [Wiki](https://github.com/microsoft/DirectXShaderC
 Visit the [DirectX Landing Page](https://devblogs.microsoft.com/directx/landing-page/) for more resources for DirectX developers.
 
 ## Downloads
-You can download the latest successful build's artifacts (built by Appveyor) for the master branch:
+You can download the latest successful build's artifacts (built by Appveyor) for the nain branch:
```
Comment (gracejennings) [inline README.md:12]: Just a comment but I think this should be "main"

Comment (hekota) [reply README.md:12]: Thanks Grace! :)

**Review by gracejennings — APPROVED** [review]
LGTM!

**Review by tex3d — APPROVED** [review]

**Comment by llvm-beanz** [comment]
Woo! Thank you @hekota! So happy to see this 🥳

**Pull request merged**